topo/node/srl/srl.go

```diff
@@ -133,26 +131,31 @@ func (n *Node) CreateNodeResource(ctx context.Context, ni node.Interface) error
 		}
 	}
 
-	log.Infof("Created Srlinux resource: %s", n.pb.Name)
+	log.Infof("Created Srlinux resource: %s", n.Name())
 
 	return nil
 }
 
-func (n *Node) DeleteNodeResource(ctx context.Context, ni node.Interface) error {
-	log.Infof("Deleting Srlinux node resource %s", n.pb.Name)
+func (n *Node) Delete(ctx context.Context) error {
+	log.Infof("Deleting Srlinux node resource %s", n.Name())
 
-	c, err := srlclient.NewForConfig(ni.RESTConfig())
+	c, err := srlclient.NewForConfig(n.RestConfig)
 	if err != nil {
 		return err
 	}
 
-	err = c.Srlinux(ni.Namespace()).Delete(ctx, n.pb.Name, metav1.DeleteOptions{})
+	err = c.Srlinux(n.Namespace).Delete(ctx, n.Name(), metav1.DeleteOptions{})
 	if err != nil {
 		return err
 	}
 
-	log.Infof("Deleted custom resource: %s", n.pb.Name)
-
+	log.Infof("Deleted custom resource: %s", n.Name())
```

Comment:
Should this log go here or after the next two deletion calls?

Reply:
done